### ISSUE TITLE
Nodetool status does not properly indicate status in an IPv6 cluster.

### ISSUE THREAD

**Issue by lseelenbinder** [scylladb/scylla#5722]
This is Scylla's bug tracker, to be used for reporting bugs only.
If you have a question about Scylla, and not a bug, please ask it in
our mailing-list at scylladb-dev@googlegroups.com or in our slack channel.

- [x] I have read the disclaimer above, and I am reporting a suspected malfunction in Scylla.

*Installation details*
Scylla version (or git commit hash): `3.2.1-0.20200122.e3e301906d5-1`
Cluster size: 3
OS (RHEL/CentOS/Ubuntu/AWS AMI): Ubuntu 18.04

I've setup an IPv6-only cluster, and it appears `nodetool status` isn't properly configured to show status for that kind of cluster. Here's the output I'm seeing:

```
$ nodetool status
Datacenter: london
==================
Status=Up/Down
|/ State=Normal/Leaving/Joining/Moving
--  Address                    Load       Tokens       Owns    Host ID                               Rack
?N  2a01:7e00:xxxx             ?          256          ?       null                                  rack0
?N  2a01:7e00:yyyy             ?          256          ?       null                                  rack0
?N  2a01:7e00:zzzz             ?          256          ?       null                                  rack0

Note: Non-system keyspaces don't have the same replication settings, effective ownership information is meaningless
```

Each of these nodes is up and healthy by all appearances. Running the following query seems to indicate it's just an issue with nodetool:

```
cqlsh:system> select peer, data_center, host_id, rack, release_version from peers   ;

 peer               | data_center | host_id                              | rack  | release_version
--------------------+-------------+--------------------------------------+-------+-----------------
 2a01:7e00::00:xxxx |      london | 52d3763f-cb46-4751-aa51-527c23a4e018 | rack0 |           3.0.8
 2a01:7e00::00:yyyy |      london | 1b6ad6c8-5b1f-4844-851a-fb627adf66be | rack0 |           3.0.8
```

Are there anymore debugging steps I should take to help you identify the issue?

**Comment by slivne** [comment]
@penberg AFAIK you fixed a similar issue on enterprise yet I am not sure this was merged to master - can you please check

**Comment by slivne** [comment]
@penberg ping

**Comment by slivne** [comment]
@penberg ping

**Comment by lseelenbinder** [comment]
Just FYI, this now works in 3.3.

**Comment by penberg** [comment]
This was fixed in commit c11874d8516365145b7f2b42ee97f69e89199cb9, which indeed is part of 3.3, but not 3.2.

**Comment by penberg** [comment]
And sorry for not updating this issue earlier. Thanks for reporting back, @lseelenbinder!
